spec/features/log_in_spec.rb

```diff
@@ -0,0 +1,8 @@
+feature 'Log in page' do
```

Comment:
Seems obvs but I lost my shit a little and forgot to slit the tests up (TBH only got to CR) but this is lovely - nice and clear separation !!